### ISSUE TITLE
Aldosterone-to-renin ratio

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7772]
**Preferred term label:**
Aldosterone-to-renin ratio

**Synonyms**
Aldosterone:renin ratio
 (ARR)

**Definition (free text, please give PubMed ID)**
Aldosterone-renin ratio (ARR) is the most reliable method for screening for primary aldosteronism. The ARR is calculated as the ratio of the serum aldosterone (in ng/dL) divided by serum plasma renin activity (in ng/mL/hour). 

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
617027 

PMID: 25907736 5/5

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000318
- label: Elevated aldosterone:renin ratio
- parents: [Abnormality of renin-angiotensin system (HP:0000847)](https://hpo.jax.org/app/browse/term/HP:0000847)
- pmid: PMID:25907736
- def: The ratio of aldosterone concentration to renin concentration is above the upper limit of normal. If aldosterone is expressed in ng/dl and plasma renin activity is expressed as ng/ml/hr, then values above 20 with aldosterone level above 15 are considered indicative of primary aldosteronism.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7772
- synonyms:
